src/main/java/core/basesyntax/dao/FruitDao.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Fruit;
+import java.util.Optional;
+
+public interface FruitDao {
+    public Optional<Fruit> get(String name);
```

Comment:
we don't need to specify public access modifier here, all abstract interface methods are public by default (same in other places)